bears/general/AnnotationBear.py

```diff
@@ -88,66 +95,130 @@ def find_annotation_ranges(self,
             A dictionary containing the various ways to define multi-line
             comments in a language.
```

Comment:
@aptrishu Encapsulating data. So that you don't have to always pass several parameters around